include/swift/AST/Module.h

```diff
@@ -744,7 +748,7 @@ class FileUnit : public DeclContext {
   // Only allow allocation of FileUnits using the allocator in ASTContext
   // or by doing a placement new.
   void *operator new(size_t Bytes, ASTContext &C,
-                     unsigned Alignment = alignof(FileUnit));
+                     unsigned Alignment = alignof(FileUnit&));
```

Comment:
Okay, then in that case we can use `alignof(FileUnit&)` in `alignOfFileUnit` and we should get the correct answer.

(I think this is actually important because DeclContexts are deliberately overaligned, and we rely on that to cram things in the low bits.)